crates/polars-ops/src/chunked_array/list/namespace.rs

```diff
@@ -255,34 +256,99 @@ pub trait ListNameSpaceImpl: AsList {
         }
     }
 
+    fn eval_lambda_on_amort(
+        s_ref: &Series,
+        lambda_expression: Arc<LambdaExpression>,
+        index_data: Option<&mut (MutablePrimitiveArray<i32>, AmortSeries)>,
+    ) -> PolarsResult<Series> {
+        let s_len = s_ref.len() as i32;
```

Comment:
What guarantee we have that s_ref.len() doesn't exceed i32::MAX ?

Reply:
Polars does not allow this as IdxSize is i32, if we ever encounter this issue(which is beyond unlikely, as we'll run out of memory first), will have to enable the big idx feature